types/k6/test/browser.ts

```diff
@@ -1203,105 +1203,105 @@ async function test() {
-    request.then(r => r?.timing());
+    // $ExpectType ResourceTiming
+    request.timing();
 
     //
     // Response
     //
-    const response = page.goto(url);
+    const response = await page.goto(url).then(r => r!);
 
-    // $ExpectType Promise<Record<string, string> | undefined>
-    response.then(r => r?.allHeaders());
+    // $ExpectType Promise<Record<string, string>>
+    response.allHeaders();
 
-    // $ExpectType Promise<ArrayBuffer | undefined>
-    response.then(r => r?.body());
+    // $ExpectType Promise<ArrayBuffer>
+    response.body();
 
-    // $ExpectType Promise<Frame | undefined>
-    response.then(r => r?.frame());
+    // $ExpectType Frame
+    response.frame();
 
-    // $ExpectType Promise<Record<string, string> | undefined>
-    response.then(r => r?.headers());
+    // $ExpectType Record<string, string>
+    response.headers();
 
-    // $ExpectType Promise<{ name: string; value: string; }[] | undefined>
-    response.then(r => r?.headersArray());
+    // $ExpectType Promise<{ name: string; value: string; }[]>
+    response.headersArray();
 
     // @ts-expect-error
-    response.then(r => r?.headerValue());
-    // $ExpectType Promise<string | null | undefined>
-    response.then(r => r?.headerValue("content-type"));
+    response.headerValue();
+    // $ExpectType Promise<string | null>
+    response.headerValue("content-type");
 
     // @ts-expect-error
-    response.then(r => r?.headerValues());
-    // $ExpectType Promise<string[] | undefined>
-    response.then(r => r?.headerValues("content-type"));
+    response.headerValues();
+    // $ExpectType Promise<string[]>
+    response.headerValues("content-type");
 
     // $ExpectType Promise<any>
-    response.then(r => r?.json());
+    response.json();
 
-    // $ExpectType Promise<boolean | undefined>
-    response.then(r => r?.ok());
+    // $ExpectType boolean
+    response.ok();
 
-    // $ExpectType Promise<Request | undefined>
-    response.then(r => r?.request());
+    // $ExpectType Request
+    response.request();
 
-    // $ExpectType Promise<SecurityDetailsObject | null | undefined>
-    response.then(r => r?.securityDetails());
+    // $ExpectType Promise<SecurityDetailsObject | null>
+    response.securityDetails();
 
-    // $ExpectType Promise<{ ipAddress: string; port: number; } | null | undefined>
-    response.then(r => r?.serverAddr());
+    // $ExpectType Promise<{ ipAddress: string; port: number; } | null>
+    response.serverAddr();
 
-    // $ExpectType Promise<number | undefined>
-    response.then(r => r?.status());
+    // $ExpectType number
+    response.status();
 
-    // $ExpectType Promise<string | undefined>
-    response.then(r => r?.statusText());
+    // $ExpectType string
+    response.statusText();
 
-    // $ExpectType Promise<{ body: number; headers: number; } | undefined>
-    response.then(r => r?.size());
+    // $ExpectType Promise<{ body: number; headers: number; }>
+    response.size();
```

Comment:
This function did not have a counter-part in Playwright, but I assumed that it's signature would be the same as the one in `Request`.